logic/src/androidMain/kotlin/com/wire/kalium/logic/network/NetworkStateObserverImpl.kt

```diff
@@ -76,14 +78,46 @@ internal actual class NetworkStateObserverImpl(
 
         val callback = object : ConnectivityManager.NetworkCallback() {
 
-            override fun onCapabilitiesChanged(network: Network, networkCapabilities: NetworkCapabilities) {
+            override fun onCapabilitiesChanged(
+                network: Network,
+                networkCapabilities: NetworkCapabilities
+            ) {
                 super.onCapabilitiesChanged(network, networkCapabilities)
-                kaliumLogger.i(
-                    "${NetworkStateObserver.TAG} capabilities changed " +
-                            "internet:${networkCapabilities.hasCapability(NetworkCapabilities.NET_CAPABILITY_INTERNET)} " +
-                            "validated:${networkCapabilities.hasCapability(NetworkCapabilities.NET_CAPABILITY_VALIDATED)}"
-                )
-                defaultNetworkDataStateFlow.update { DefaultNetworkData.Connected(network, networkCapabilities) }
+                val loggerMessage = buildString {
```

Comment:
Suggestion: We can try to log in `json` format, so we can later parse it and do some filtering in Datadog

You can take a look at this utility:

https://github.com/wireapp/kalium/blob/a2f0eadaaa0e9790b97b5aa35892749eb3bfb879/logic/src/commonMain/kotlin/com/wire/kalium/logic/CoreLogger.kt#L63

Reply:
thanks, added. Good to know we such method